.github/workflows/macos.yml

```diff
@@ -0,0 +1,431 @@
+              echo "JAVA_HOME=$JAVA_HOME_17_X64" >> $GITHUB_ENV
+              echo "JAVA_HOME = $JAVA_HOME"
+              echo "/usr/lib/jvm"
+              ls -l /usr/lib/jvm
+              echo "/usr/local/lib/jvm"
+              ls -l /usr/local/lib/jvm
+              ./scripts/install_linux.sh
+            fi
+            ;;
+          macOS)
+            if [ "$CHECK_REUSE" != "ONLY" -a "$CHECK_MKVK" != "ONLY" ]; then
+              if [ "$FEATURE_DOC" = "ON" ]; then
+                  brew install doxygen
+                  brew install graphviz
+              fi
+              env
+              ./scripts/install_macos.sh
+            fi
+            ;;
+          esac
+
+      - name: before_script
+        run: |
+          git fetch --unshallow --tags
+          case "${RUNNER_OS:-Linux}" in
+          Linux)
+            ;;
+          macOS)
+            ./scripts/before_build_macos.sh
+            ;;
+          esac
+          if [ "$FEATURE_TOOLS_CTS" = "ON" ]; then
+            git submodule update --init --recursive tests/cts
+          fi
+          # Make sure embedded dates are correct.
+          ./install-gitconfig.sh
+          scripts/smudge_date.sh
+
+      - name: script
+        run: |
+          set -e # Instead of trying to chain everything together with &&
+                 # otherwise subsequent commands will swallow bad exit status.
+          case "${RUNNER_OS:-Linux}" in
+          Linux)
+            if [ "$CHECK_REUSE" = "YES" -o "$CHECK_REUSE" = "ONLY" ]; then
+              echo "Calling reuse lint."
+              reuse lint
+            fi
+            if [ "$CHECK_REUSE" != "ONLY" -a "$CHECK_MKVK" != "ONLY" ]; then
+              if [ "$WASM_BUILD" = "YES" ]; then
+                ./scripts/build_wasm_docker.sh
+              else
+                ./scripts/build_linux.sh
+              fi
+            fi
+            ;;
+          macOS)
+            if [ "$CHECK_REUSE" != "ONLY" -a "$CHECK_MKVK" != "ONLY" ]; then
+              if [ "$PLATFORM" = "macOS" ]; then
+                ./scripts/build_macos.sh
+              else
+                ./scripts/build_ios.sh
+              fi
+            fi
+            ;;
+          esac
+          if [ "$CHECK_MKVK" = "ONLY" ]; then
+            scripts/check_mkvk.sh
+          fi
+          if [ "$DEPLOY_DOCS" = "YES" ]; then
+            # Some files in the pyktx docs have an _ prefix so Jekyll will
+            # not copy them from the gh-pages branch to the website. This
+            # file says no Jekyll files here. Treat all as ordinary files.
+            touch $BUILD_DIR/docs/html/.nojekyll
+          fi
+          set +e
+
+      - name: before_deploy
+        run: |
+          gem install octokit:'~> 4.15.0' mime-types
+          case "${RUNNER_OS:-Linux}" in
+          Linux)
+            ;;
+          macOS)
+            # Only notarize if we're going to deploy. I.e on a tag. Also check for
```

Comment:
Move this so notarization is done for every package that will be saved as an artifact of the build. It was limited to deployment on Travis because no artifacts were saved.